sample-apps/dotnet/dotnet-ec2-win-main-setup.ps1

```diff
@@ -0,0 +1,99 @@
+param (
+    [string]$GetCloudwatchAgentCommand,
+    [string]$GetAdotDistroCommand,
+    [string]$GetSampleAppCommand,
+    [string]$TestId,
+    [string]$RemoteServicePrivateEndpoint,
+    [string]$TestCanaryType,
+    [string]$AWSRegion
+)
+
+# This file is used to deploy and instrumentation main sample app for Dotnet E2E Canary test
+# This is the most stable way to do that in automatically test workflow invloving EC2 and SSM
+
+# To avoid written UI for download and extract zip step, saving lots of time
+$ProgressPreference = 'SilentlyContinue'
+
+# Install Dotnet
+wget -O dotnet-install.ps1 https://dot.net/v1/dotnet-install.ps1
+.\dotnet-install.ps1 -Version 8.0.302
+
+# Install and start Cloudwatch Agent
+Invoke-Expression $GetCloudwatchAgentCommand
+
+Write-Host "Installing Cloudwatch Agent"
```

Comment:
Do these logs show up somewhere useful? I can't find them in the workflow runs provided in the PR description

Reply:
Yes, it shows in SSM run command history on the AWS console

Reply:
Let's include that in the runbook